### PR TITLE
dev-4817 updated documentation to not mention old functionality

### PR DESCRIPTION
**Pull request by BrianZito**
**Description:**
Updates documentation:

* Don't mention the old `agency` param, as that's now legacy
* Do add toptier_name to instances of the agency object

### CODE REVIEW

**Review by sethstoudenmier — APPROVED** [review]
LGTM

File: usaspending_api/api_contracts/contracts/v2/bulk_download/awards.md [hunk 1]
```diff
@@ -90,10 +90,7 @@ This route sends a request to the backend to begin generating a zipfile of award
 ## Filter Objects
 
 ### Filters (object)
-+ `agencies` (optional, array[Agency], fixed-type)
-All requests must either include the `agencies` or `agency` parameter.
-+ `agency` (optional, string)
```
Comment (sethstoudenmier) [inline usaspending_api/api_contracts/contracts/v2/bulk_download/awards.md:95]: Should also be able to remove `+ sub_agency (optional, string)`

**Review by sethstoudenmier — DISMISSED** [review]

**Review by sethstoudenmier — COMMENTED** [review]
Can you also remove `agency` from the sample request body?

**Review by sethstoudenmier — APPROVED** [review]

File: usaspending_api/api_contracts/contracts/v2/bulk_download/awards.md [hunk 2]
```diff
@@ -24,7 +24,6 @@ This route sends a request to the backend to begin generating a zipfile of award
 
             {
                 "filters": {
-                    "agency": 50,
```
Comment (tony-sappe) [inline usaspending_api/api_contracts/contracts/v2/bulk_download/awards.md:27]: Please include a valid JSON example

image

**Review by tony-sappe — CHANGES_REQUESTED** [review]

File: usaspending_api/api_contracts/contracts/v2/bulk_download/awards.md (hunk 1 above)
Comment (tony-sappe) [reply usaspending_api/api_contracts/contracts/v2/bulk_download/awards.md:95]: @BrianZito is it valid to remove this filter from the API contract? If not a brief response would be appreciated

Comment (BrianZito) [reply usaspending_api/api_contracts/contracts/v2/bulk_download/awards.md:95]: Good catch

**Review by tony-sappe — APPROVED** [review]

**Pull request merged**